src/docs/pages/grid/GridTemplateAreas.tsx

```diff
@@ -0,0 +1,78 @@
+import { VuiGrid, VuiGridItem, VuiPanel, VuiSpacer, VuiText } from "../../../lib";
+
+export const GridTemplateAreas = () => (
+  <>
+    <VuiText>
+      <p>Use `templateAreas` to set grid-template-areas.</p>
+      <p>grid-template-areas is a string-based grid area template.</p>
+      <ul>
+        <li>Each quoted string is a row in a 2D matrix.</li>
+        <li>Within a row, space-separated identifiers name the cells (e.g., header, sidebar).</li>
+        <li>Repeating the same identifier across adjacent cells merges them into one named grid area.</li>
+        <li>A single dot . denotes an unassigned cell.</li>
+        <li>All rows must have the same number of tokens.</li>
+        <li>
+          Elements join an area with grid-area: <code>grid-area: &lt;identifier&gt;.</code>
+        </li>
+      </ul>
+
+      <p>Example:</p>
+      <pre>
+        <code>
+          {`grid-template-areas: 
+\t"header header header"
+\t"sidebar content content"
+\t"sidebar content content"
```

Comment:
Per my earlier comment, I don't understand what the row-spanning is doing in this example. If we can't make its function clearer, let's remove it.